frontend/src/services/like.service.ts

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import {NEST_URI} from '@env'
+
+import {getAsyncItem, setAsyncItem} from "./async-storage.service";
+import {deleteItemAsync} from "expo-secure-store";
+
+
+type LikeDislike = {
+    tagName: string,
+    count: number
+}
+
+
+export const addLike = async (id: string | undefined, like: any) => {
+    const likeDislike: string | null = await getAsyncItem('likeDislike');
+    if (!likeDislike) {
+        await setAsyncItem('likeDislike', JSON.stringify([]));
+    }
+    const parsedLikeDislike: LikeDislike[] = JSON.parse(likeDislike || "[]");
+    const accessToken = await getAsyncItem('spotify_access_token');
+    axios.post(`${NEST_URI}/likes/like/:${id}`, like, {
+        headers: {
+            Authorization: accessToken
+        }
+    }).then(async res => {
+        const genres = res.data.data;
+        genres.forEach(genre => {
+            const existingGenre = parsedLikeDislike.find(likeDislikeGenre => likeDislikeGenre.tagName === genre);
+            if (!existingGenre) {
+                parsedLikeDislike.push({tagName: genre, count: 1});
+            } else {
+                existingGenre.count++;
+            }
+        });
+        await setAsyncItem('likeDislike', JSON.stringify(parsedLikeDislike));
+        console.log(await getAsyncItem('likeDislike'));
```

Comment:
@Imadbk7 I think you forgot to remove your console logs as they are used for debugging purposes.